### PR TITLE
xattr bug

### PR DESCRIPTION
**Pull request by jief666**
Here is a little correction about xattr attributes.

(copy and paste !!!, I do that all the time :-)

### CODE REVIEW

File: src/main-fuse.cpp [hunk 1]
```diff
@@ -294,6 +294,10 @@ int hfs_listxattr(const char* path, char* buffer, size_t size)
 		for (const std::string& str : attrs)
 			output.insert(output.end(), str.c_str(), str.c_str() + str.length() + 1);
 
+        if ( buffer == NULL ) {
```
Comment (bugaevc) [inline src/main-fuse.cpp]: This file is indented with tabs, not 8 spaces, and conditions are formatted `(like this)`, not `( like this )`.

Comment (jief666) [reply src/main-fuse.cpp]: Sorry, my editor convert tabs, I guess. "( like this )" this is habit. Makes it easier to read.

**Comment by jief666** [comment]
Hi, will you consider taking that pull request, or is it a no-go because of the format ?
(I have few more waiting, that's why I ask).
Thanks.

**Comment by LubosD** [comment]
I've now done manual merging of the commits while fixing the formatting via `git rebase`.

**Comment by jief666** [comment]
Fixing formatting with rebase ?? There is maybe something you can teach me there 👍 
I'll rebase onto you master for the next pull request.
Thanks.

**Comment by LubosD** [comment]
I used this:
```
git rebase --interactive somecommit^
```
then you change "pick" to "edit" for the commits you want to change, start editing (+`git add`) and use `git rebase --continue` when you're done with a commit.

**Pull request closed without merging**